.github/workflows/main.yml

```diff
@@ -19,7 +19,7 @@ jobs:
           curl -fsSLO https://github.com/llvm/llvm-project/releases/download/llvmorg-10.0.0/LLVM-10.0.0-win64.exe
           7z x LLVM-10.0.0-win64.exe -y -o"llvm"
           echo "$(pwd)/llvm/bin" >> $GITHUB_PATH
-          echo "WASM_AR=$(pwd)/llvm/bin/llvm-ar.exe" >> $GITHUB_ENV
+          echo "AR=$(pwd)/llvm/bin/llvm-ar.exe" >> $GITHUB_ENV
```

Comment:
Oh oops, I set `AR` twice, but it is on the path so it still works. Maybe someone tell what what style they prefer?

Reply:
Either way seems fine, but right now the other platforms set AR below, and it seems best to be consistent about it.